packages/server/lib/hooks/connection/verificatiion-script.ts

```diff
@@ -0,0 +1,137 @@
+import type { Span } from 'dd-trace';
+import type { AxiosError, AxiosResponse } from 'axios';
+import type { Config } from '@nangohq/shared';
+import { getProvider, ProxyRequest, getProxyConfiguration } from '@nangohq/shared';
+import * as verificationscriptHandlers from './index.js';
+import type {
+    InternalProxyConfiguration,
+    ApiKeyCredentials,
+    BasicApiCredentials,
+    TbaCredentials,
+    JwtCredentials,
+    SignatureCredentials,
+    UserProvidedProxyConfiguration,
+    DBConnectionDecrypted,
+    ConnectionConfig,
+    MessageRowInsert
+} from '@nangohq/types';
+import tracer from 'dd-trace';
+import { stringifyError } from '@nangohq/utils';
+
+type VerificationScriptHandler = (internalNango: InternalNango) => Promise<void>;
+
+type VerificationScriptHandlersMap = Record<string, VerificationScriptHandler>;
+
+const handlers: VerificationScriptHandlersMap = verificationscriptHandlers as unknown as VerificationScriptHandlersMap;
+
+export interface InternalNango {
+    getCredentials: () => Promise<{
+        credentials: ApiKeyCredentials | BasicApiCredentials | TbaCredentials | JwtCredentials | SignatureCredentials;
+        providerConfigKey: string;
+    }>;
+    proxy: <T = any>({ method, endpoint, data, headers, params, baseUrlOverride }: UserProvidedProxyConfiguration) => Promise<AxiosResponse<T> | AxiosError>;
+}
+
+async function execute(
+    config: Config,
+    credentials: ApiKeyCredentials | BasicApiCredentials | TbaCredentials | JwtCredentials | SignatureCredentials,
+    connectionId: string,
+    connectionConfig: ConnectionConfig
+) {
+    const { provider: providerName, unique_key: providerConfigKey } = config;
+
+    const connection: DBConnectionDecrypted = {
+        id: -1,
+        end_user_id: null,
+        provider_config_key: config.unique_key,
+        connection_id: connectionId,
+        credentials,
+        connection_config: connectionConfig,
+        environment_id: config.environment_id,
+        created_at: new Date(),
+        updated_at: new Date(),
+        config_id: -1,
+        credentials_iv: null,
+        credentials_tag: null,
+        deleted: false,
+        deleted_at: null,
+        last_fetched_at: null,
+        metadata: null
+    };
+
+    const activeSpan = tracer.scope().active();
+
+    const span = tracer.startSpan('nango.server.hooks.verificationScript', {
+        childOf: activeSpan as Span,
+        tags: {
+            'nango.provider': providerName,
+            'nango.providerConfigKey': providerConfigKey,
+            'nango.connectionId': connectionId
```

Comment:
The prefix is legacy you can remove
```suggestion
            'provider': providerName,
            'providerConfigKey': providerConfigKey,
            'connectionId': connectionId
```

Reply:
removed some probably left over ones with this [commit](https://github.com/NangoHQ/nango/pull/3666/commits/a2261a01df0d1ee9ad98930173d8928c6633891c) as well